app/utilities/crypto.py

```diff
@@ -0,0 +1,13 @@
+from hashlib import sha256
+
+#TODO how to name that folder 
```

Comment:
#TODO how to name that folder 
solve it before merging PR

Reply:
done